### PR TITLE
IN-608-[SDK]-Plugin enhance test coverage

### PR DESCRIPTION
**Pull request by bhavin-te**
# Description

This PR contains code-changes for IN-608 - Enhanced Test Coverage for Plugins[SDK].

Module : `tenable\io\plugins.py `
Statements covered by Test-cases (%) : 91%

Added test case to cover maximum statement of Plugins module. Earlier test cases covered only 74% statement of plugins module. Now 91% statement is covered.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

To test code-changes of this PR, I have run testcases of `pyTenable\tests\io\test_plugins.py`. All testcases of this file working fine and getting passed. please find attached snapshot.

TC Plugins

Also I have run all test-cases of `pyTenable\tests `and it is getting passed. please find attached snapshot.

TC plugins All

**Test Configuration**:
* Python Version(s) Tested:
* Tenable.sc version (if necessary):

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by MitaliUpadhyay — APPROVED** [review]
File changes look good to me :)

File: tests/io/test_plugins.py [hunk 1]
```diff
@@ -111,3 +111,15 @@ def test_plugins_list_success(api):
 def test_plugin_iterator_populate_family_cache(api):
     '''test for _populate_family_cache in PluginIterator'''
     getattr(PluginIterator(api), '_populate_family_cache')()
+
+
+@pytest.mark.vcr()
+def test_populate_family_cache_with_maptable(api):
```
Comment (MitaliUpadhyay) [inline tests/io/test_plugins.py:117]: Just a minor suggestion, change test-case name to test_plugins_populate_family_cache_with_maptable

**Review by alpesh-te — APPROVED** [review]
LGTM!

**Review by MitaliUpadhyay — APPROVED** [review]

File: tests/io/test_plugins.py (hunk 1 above)
Comment (bhavin-te) [reply tests/io/test_plugins.py]: Addressed comment

**Review by varap11 — APPROVED** [review]

**Pull request merged**